test/e2e/service.go

```diff
@@ -1021,8 +1025,7 @@ func testReachable(ip string, port int) {
 		Failf("got port==0 for reachability check", url)
 	}
 
-	By(fmt.Sprintf("Checking reachability of %s", url))
-	for t := time.Now(); time.Since(t) < podStartTimeout; time.Sleep(5 * time.Second) {
+	for t := time.Now(); time.Since(t) < timeout; time.Sleep(5 * time.Second) {
```

Comment:
I know you didn't write this, but could you also please switch this to standard `wait.Poll(...)` usage?